### ISSUE TITLE
Downloading stopped working today for lynda

### ISSUE THREAD

**Issue by webis** [rg3/youtube-dl#9740]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like that [x])
- Use *Preview* tab to see how your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2016.06.03*. If it's not read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x ] I've **verified** and **I assure** that I'm running youtube-dl **2016.06.03**

### Before submitting an *issue* make sure you have:
- [x ] At least skimmed through [README](https://github.com/rg3/youtube-dl/blob/master/README.md) and **most notably** [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x ] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones

### What is the purpose of your *issue*?
- [x ] Bug report (encountered problems with youtube-dl)
- [ ] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### The following sections concretize particular purposed issues, you can erase any section (the contents between triple ---) not applicable to your *issue*

---

### If the purpose of this *issue* is a *bug report*, *site support request* or you are not completely sure provide the full verbose output as follows:

Add `-v` flag to **your command line** you run youtube-dl with, copy the **whole** output and insert it here. It should look similar to one below (replace it with **your** log inserted between triple ```):
```
[debug] System config: []
[debug] User config: []
[debug] Command-line args: ['http://www.lynda.com/Photoshop-tutorials/Photoshop-
CC-One-One-Mastery/124099-2.html', '--username', 'PRIVATE', '--password', 'PRIVA
TE', '--playlist-start', '95', '--playlist-end', '103', '--verbose']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2016.06.03
[debug] Python version 3.4.4 - Windows-7-6.1.7601-SP1
[debug] exe versions: none
[debug] Proxy map: {}
[lynda:course] Logging in as david@husnian.com
ERROR: Unable to log in; please report this issue on https://yt-dl.org/bug . Mak
e sure you are using the latest version; type  youtube-dl -U  to update. Be sure
 to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpjletvtok\bu
ild\youtube_dl\YoutubeDL.py", line 676, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpjletvtok\bu
ild\youtube_dl\extractor\common.py", line 340, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpjletvtok\bu
ild\youtube_dl\extractor\common.py", line 334, in initialize
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpjletvtok\bu
ild\youtube_dl\extractor\lynda.py", line 23, in _real_initialize
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\rg3\tmpjletvtok\bu
ild\youtube_dl\extractor\lynda.py", line 81, in _login
youtube_dl.utils.ExtractorError: Unable to log in; please report this issue on h
ttps://yt-dl.org/bug . Make sure you are using the latest version; type  youtube
-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and includ
e its complete output.
```

---

### Description of your *issue*, suggested solution and other information

I've been able to download files from my paid account for many months now but today sometime (not sure when) it stopped working with the above error. I tried it on and off for some hours and it still didn't work. Note: today, prior to when it stopped working, I was able to download a video.

As side note, I did try logging into the site and I noticed that the flow was different and I suspect maybe even the URL but I can't verify that because I hadn't noticed the old URL. Anyway, they seem to have split the log in into 2 screens instead of 1 so that may have impacted your software.

Thanks.

**Comment by 1dinamani** [comment]
I also have the same error. I was able to download Yesterday, today i am not able to download. I have received exactly same error. Please assist.

**Comment by 1dinamani** [comment]
Hi dstftw you have closed both the ticket. But not received any solution for it. # 9743 & #9740

Please asssit.

**Comment by dstftw** [comment]
I did not close this issue. There is no solution currently.

**Comment by dstftw** [comment]
Fixed in 2016.06.11.1.
